src/QMCWaveFunctions/BsplineFactory/BsplineSetT.h

```diff
@@ -245,5 +245,8 @@ class BsplineSetT : public SPOSetT<T>
   std::vector<int> offset;
 };
 
+extern template class BsplineSetT<float>;
```

Comment:
Is this needed?

Reply:
I had thought they were because of some build errors but I'm not seeing them after removing these lines so I guess not, though I am seeing some unrelated build errors from `LCAOrbitalBuilderT`